build/configs/imxrt1020-evk/hello/defconfig

```diff
@@ -383,7 +383,7 @@ CONFIG_ARCH_BOARD_HAVE_FLASH=y
 CONFIG_FLASH_PARTITION=y
 CONFIG_FLASH_MINOR=0
 CONFIG_FLASH_PART_SIZE="2048,512,512,4096,"
-CONFIG_FLASH_PART_TYPE="none,ftl,config,smartfs,"
+CONFIG_FLASH_PART_TYPE="none,ftl,config,jSmartFs,"
```

Comment:
For device configurations like these which support a second flash and partitions on the same,
The defconfig partition name for the second flash also needs to be changed.
The same is required to be done for Mediatek mt7686_hdk defconfigs in RS6 please.